### ISSUE TITLE
Revise when the leg is considered on the rug

### ISSUE THREAD

**Issue by phet-steele** [phetsims/john-travoltage#241]
Positions -4 and 3 should probably not be considered "rubbing on" the rug. Visually they might be on the rug, sure, but as far as the sim is concerned these positions do not qualify to grab an electron. For example, place the leg at position -4 and increment towards -3 (which is further onto the rug) and you will not pick up an electron. Go back and forth between these two positions as much as you want and you will never get an electron (same goes with sliding between 2 and 3). However, moving between -3 and -2 will *always* give you an electron (hopefully 😄). This is the reason -3 should continue to say "rubbing on" and -4 should not.

If we are using the paradigm of "rubbing on" the carpet meaning the leg is ready to grab electrons, then it is false to say that positions -4 and 3 are rubbing. You can never travel to or from these positions and grab an electron.

Seen on macOS 10.12.4 Safari. For phetsims/tasks/issues/831.

**Comment by jessegreenberg** [comment]
Good point @phet-steele I am inclined to agree.  The down side to this is that there will be more positions off of the rug, which is one of the things that we addressed in #222.  But 5 positions max doesn't seem too bad.

Should this be changed @emily-phet?

**Comment by emily-phet** [comment]
@jessegreenberg I agree with @phet-steele. I thought this was the case, but wasn't sure that you could *never* grab an electron at some of these edge positions. If it's the case that moving to position -4 or position 3 never pick up an electron, these should be described as off the rug.

**Comment by jessegreenberg** [comment]
Thanks @emily-phet, I will change - assigning to me.

**Comment by jessegreenberg** [comment]
Done in the above commit.  I don't think that this needs to be reviewed, the change was simple enough. Here is proof:

<img width="808" alt="screen shot 2017-05-05 at 1 44 36 pm" src="https://cloud.githubusercontent.com/assets/6396244/25761832/73529de8-3199-11e7-8950-84d780004594.png">
<img width="815" alt="screen shot 2017-05-05 at 1 44 52 pm" src="https://cloud.githubusercontent.com/assets/6396244/25761833/74854eea-3199-11e7-965a-cb0be8eb2121.png">


@phet-steele if you disagree, feel free to reopen.
